app/src/main/res/layout/item_status.xml

```diff
@@ -108,23 +108,42 @@
             android:textColor="?android:textColorPrimary"
             android:textSize="?attr/status_text_medium" />
 
-        <ToggleButton
-            android:id="@+id/status_content_warning_button"
+        <RelativeLayout
```

Comment:
It is now possible to ged rid of the `FlowLayout`, and the two `RelativeLayout`s are unnecessary as well. Please align the button within the toplevel `RelativeLayout`

Reply:
I'll try removing the `FlowLayout`.
The extra `RelativeLayout` is there because the centering didn't work correctly unless I nested them like this.